source/includes/monitoring/sdam.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+require __DIR__ . '/vendor/autoload.php';
+
+// start-mysubscriber
+class MySubscriber implements MongoDB\Driver\Monitoring\SDAMSubscriber
+{
+    private $stream;
+    public function __construct($stream)
+    {
+        $this->stream = $stream;
+    }
+    public function serverOpening(MongoDB\Driver\Monitoring\ServerOpeningEvent $event): void
```

Comment:
```suggestion

    public function serverOpening(MongoDB\Driver\Monitoring\ServerOpeningEvent $event): void
```

Also, between any methods with bodies.